Content.Shared/Doors/Components/DoorComponent.cs

```diff
@@ -302,6 +302,9 @@ public sealed partial class DoorComponent : Component
 
     [DataField(customTypeSerializer: typeof(ConstantSerializer<DrawDepthTag>))]
     public int ClosedDrawDepth = (int) DrawDepth.DrawDepth.Doors;
+
+    [DataField, AutoNetworkedField]
+    public bool Powered;
 }
```

Comment:
Doors don't require power, e.g. stone doors are valid. Adding this field to 3 comps is kinda nasty.